src/lib.rs

```diff
@@ -138,6 +138,7 @@ pub fn init_and_run(options: Options) -> ! {
 
     let window = winit::window::WindowBuilder::new()
         .with_title(BASE_WINDOW_TITLE)
+        .with_inner_size(winit::dpi::LogicalSize::new(1280, 720))
```

Comment:
why do we need this?

Reply:
This is very interesting and I learned about this just recently (when contributing a fix to winit about `WindowBuilder::with_maximized`). On Windows, and perhaps other platforms as well, "maximized" is only a state. When that state is exited, the window returns to the size it previously had. By specifying a size, we can ensure that size is something usable.

Reply:
So specifying both `with_maximized` and `with_inner_size` actually has meaning.